### ISSUE TITLE
In Scan Mode, Screen Reader announces Separator as Checkbox Instead of Separator:A11y_Visual Studio Code Client_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#282663]
## GitHub Tags:
#A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM_Visual Studio Code Client_Win32_JULY2024; #DesktopApp; #Win32; #Benchmark;#Name Role Value;#MAS4.1.2; #Closed; #A11yFixed;

## Environment and OS details:
Application Name: Visual Studio Code
Version: 1.107.0 (user setup)
Win11 25H2 OS build :26100.4770

## Reproduction Steps:
1. Launch VS Code 
2. Navigate to 'Explorer' tab from left navigation 
3. Access 'Open Editors' section and activate 'New File' button 
4. Open context menu (Ctrl + Shift + F10)  for newly added file
5. Press down arrow key to Reopen editor with  and press ENTER key.
6. Press down arrow key to Jupyter Notebook and press ENTER key.
7. TAB till More option button code cell and press ENTER key.
8. Now navigate in scan mode to separator with down arrow key and observe the issue

## Actual Result:
While navigating in Scan Mode using a screen reader, the focus moves to a separator element, but it is incorrectly announced as a checkbox. 

## Expected Result:
When the screen reader focus lands on a separator, it should be correctly announced as a "separator", not as a different element type like a checkbox or screen reader focus should not move on separator in scan mode

## User Impact:
Visually impaired users may get confused due to incorrect information about the UI element. This can lead to misunderstandings about the page structure and reduce overall accessibility. 

## Attachment:

<img width="1365" height="735" alt="Image" src="https://github.com/user-attachments/assets/2d88343c-c203-49a6-82b9-64b73288a216" />

Code snippet

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/da6c4cb7-2e91-4715-b813-0bba4c11d483" />
